lib/Github/Api/Repository/Actions/Workflows.php

```diff
@@ -24,30 +24,48 @@ public function all(string $username, string $repository, array $parameters = []
     }
 
     /**
-     * @link https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#get-a-workflow
+     * @link https://docs.github.com/en/rest/reference/actions#get-a-workflow
      *
      * @param string $username
      * @param string $repository
-     * @param int    $workflowId
+     * @param string $workflow
      *
      * @return array
      */
-    public function show(string $username, string $repository, int $workflowId)
+    public function show(string $username, string $repository, string $workflow)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.$workflowId);
+        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.rawurlencode($workflow));
     }
 
     /**
-     * @link https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#get-workflow-usage
+     * @link https://docs.github.com/en/rest/reference/actions#get-workflow-usage
      *
      * @param string $username
      * @param string $repository
-     * @param int    $workflowId
+     * @param string $workflow
      *
      * @return array|string
      */
-    public function usage(string $username, string $repository, int $workflowId)
+    public function usage(string $username, string $repository, string $workflow)
```

Comment:
This is actually a BC break, as users who pass an integer will get a TypeError now. Can you remove the string typehint so we allow both int and string values (but add a `@param` doc for this parameter to defined the allowed types). And check in the method itself if it's a string, then rawurlencode the value otherwise just pass it to the url.